Automattic/jetpack
# Fix SEO tools visibility on Simple/Atomic sites

Fixes https://github.com/Automattic/wp-calypso/issues/84639

## Proposed changes:

As mentioned in the original issue, the SEO tools show up on Free/Premium tiers which shouldn't be the case. We only should show this functionality on Business plan and higher. 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Use rsync or the cli command in order to copy changes from this PR to your sandbox.
- Modify your hosts file to point your simple/atomic site to your sandbox
- Create a Free or Premium site. Create a new Post. In Jetpack right sidebar, **no** SEO tools should be visible.
- Create a Business Plan. Create a new Post. In Jetpack right sidebar, SEO tools should be visible.
